test/screenshotter/ss_data.yaml

```diff
@@ -107,7 +107,7 @@ KaTeX: \KaTeX
 Kern:
     tex: \frac{a\kern{1em}b}{c}a\kern{1em}b\kern{1ex}c\kern{-0.25em}d
     nolatex: LaTeX fails to typeset this, “Missing number, treated as zero.”
-Lap: ab\llap{f}cd\rlap{g}h
+Lap: ab\mathllap{f}cd\mathrlap{g}h
```

Comment:
We should probably also verify the changed behavior of `llap` et al.

Reply:
I'll add some tests.